h5py/_hl/files.py

```diff
@@ -411,3 +416,7 @@ def __repr__(self):
         if six.PY2:
             return r.encode('utf8')
         return r
+
+
+register_driver('fileobj',
+                lambda plist: plist.set_driver(h5fd.fileobj_driver))
```

Comment:
Shouldn't this been in h5py/h5fd.pyx?

Reply:
I'm not sure -- `register_driver()` was already defined in https://github.com/h5py/h5py/blob/master/h5py/_hl/files.py and seems perfectly suitable.

Reply:
Do you mean extending `_drivers` init value rather than using `register_driver` call? Yes, I can certainly make such a mini-refactoring.

Reply:
Yeah, might as well keep all the builtins in one place.